#!/bin/bash
#==========================================================================
# CFTR2 T1K Reference Sequence Extraction and Analysis
#==========================================================================
#
# STEP 0: PREREQUISITE FILES
#
#   1. Install T1K:
#         Clone the T1K repository from:
#         https://github.com/mourisl/T1K.git
#
#   2. Reference Allele Sequences:
#      Ensure that the allele reference sequences generated by the preceding
#      CFTR2 Variant Integration and Ensembl Formatter Pipeline are available:
#         - CFTR2_rna.fa (for RNA-seq data)
#         and/or
#         - CFTR2_dna.fa (for WES and WGS data)
#
#==========================================================================
#
# STEP 1: RUN T1K ANALYSIS
#
# This step executes the T1K pipeline to genotype CFTR2.
# T1K performs variant calling using paired-end FASTQ files and a provided reference
# allele sequence.
#
# To genotype CFTR2 on RNA-seq data
#   ./run-t1k -1 read_1.fq -2 read_2.fq -f CFTR2_rna.fa 
#
# To genotype CFTR2 on whole genome sequencing data
#   ./run-t1k -1 read_1.fq -2 read_2.fq -f CFTR2_dna.fa 
#
# Ensure the appropriate reference file (CFTR2_rna.fa or CFTR2_dna.fa) is available
# in the working directory before running T1K.
#
#==========================================================================
